CMakeLists.txt

```diff
@@ -1,7 +1,8 @@
 project(co2mon)
 cmake_minimum_required(VERSION 2.8)
 
-set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} -Wall -Wextra -Wno-unused-parameter")
```

Comment:
You are removing -Wno-unused-parameter and, at the same time, you are introducing UNUSED macros.